components/NewFooter /styles.module.scss

```diff
@@ -0,0 +1,24 @@
+.row {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 14px;
+}
+
+.container {
+    border-top: 1px solid grey;
+    margin: 30px 0px;
+}
+
+.text {
+    padding: 0px 100px;
```

Comment:
Remove padding. Spacing should be handled using flexbox css properties (i.e. `justify-content`, `align-items`)